esgf-technical-overview.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: default
+title: ESGF Technical Overview
+---
+
+## ESGF Technical Overview
+
+This document gives an architectural overview of the technical components of
+The Earth System Grid Federation (ESGF). This may be a useful reference for
+understanding the relationship between components of an installation, as well
+as providing information about the purpose of each component and links to
+relevant documentation pages and installation guides.
+
+For a more general description of the purpose and high-level structure of the
+federation, see [ESGF Federation Design].
+
+*(A good place for a diagram)*
```

Comment:
We should discuss @watucker with others which components should be present in the diagram.  Once we have that,  I think we can figure out how they are connected.  Maybe do this during the 8 Nov CDNOT or 9 Nov Sprint meetings?

Reply:
Perhaps just the components listed below?  we can clarify if anything else